stackslib/src/chainstate/burn/db/processing.rs

```diff
@@ -135,17 +135,13 @@ impl<'a> SortitionHandleTx<'a> {
         let total_burn = state_transition
             .accepted_ops
             .iter()
-            .fold(Some(0u64), |acc, op| {
```

Comment:
Is this from clippy, or is this something you added?  Because I don't know if these are semantically equivalent.

Reply:
Yes Clippy warned about this, but I had to make the change manually. These are not quite equivalent, `try_fold()` will quit when it finds `None`, whereas the `fold()` version will always run through the entire iterator. So `try_fold()` can be more efficient, but they always produce the same value

Reply:
Okay, since they aren't semantically equivalent, please revert this `try_fold` change and the one below.